### PR TITLE
feat(compare): basic compare tool functionality

### PR DESCRIPTION
**Pull request by danazkari**
### Steps for testing
- Go to block producers page
- A big dark box should appear above the list of block producers, with the title compare tool.
- Go ahead and click the `+` sign on any block producer, and then click a second one.
- Both block producers should appear in the compare tool on the same radar graph.
- Click on the switch on the bottom right corner, it should change to a horizontal slider to show individual block producers.

image

### CODE REVIEW

**Comment by gaboesquivel** [comment]
thanks @danazkari !  it looks great.  
@murillojorge take a look, pls review ux flow.

File: src/routes/all-bps/AllBps.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Redux } from 'redux-render'
+import PropTypes from 'prop-types'
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles'
+import { translate } from 'react-i18next'
+// import BPCard from 'components/bp-card/bp-card'
```
Comment (mahcr) [inline src/routes/all-bps/AllBps.js]: Remove this comment

File: src/components/compare-tool/compare-slider-view.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+CompareSliderView.propTypes = {
+  classes: PropTypes.object.isRequired,
+  t: PropTypes.func.isRequired,
+  selected: PropTypes.array.isRequired,
+  className: PropTypes.string
+}
+
+CompareSliderView.defaultProps = {}
+
+export default withStyles(style)(translate('translations')(CompareSliderView))
```
[the remark this answers is not in the document]
Comment (danazkari) [reply src/components/compare-tool/compare-slider-view.js]: Done! Thanks for checking!

File: src/routes/all-bps/AllBps.js (hunk 1 above)
Comment (danazkari) [reply src/routes/all-bps/AllBps.js]: Removed the entire file, we weren't using it anyways.

**Pull request merged**